frontend/src/components/DonutCartCard/DonutCartCard.tsx

```diff
@@ -1,35 +1,43 @@
-import React, { FC } from 'react';
+import React, { FC, useState } from 'react';
 import './DonutCartCard.css';
 import { Card, Button, InputGroup, FormControl, Form } from 'react-bootstrap';
 import donut from '../../assets/chocolate_glaze.jpg';
+import { Donut } from '../../types';
 
-interface DonutCartCardProps {}
+interface DonutCartCardProps {
+  donut : Donut
+}
 
-const DonutCartCard: FC<DonutCartCardProps> = () => (
+const DonutCartCard: FC<DonutCartCardProps> = (props : DonutCartCardProps) => {
+  const [quantity, setQuantity] = useState<number>(0);
```

Comment:
Awesome, we only need read-only pages for now but this will make it even easier in the future iterations